cms/models/placeholdermodel.py

```diff
@@ -729,6 +729,13 @@ def _recalculate_plugin_positions(self, language):
             sql = sql.format(connection.ops.quote_name(CMSPlugin._meta.db_table))
             cursor.execute(sql, [self.pk, language])
         else:
-            raise RuntimeError(
-                '{} is not supported by django-cms'.format(connection.vendor)
+            sql = (
```

Comment:
The raise exception has now been removed for any other unsupported db's. This should be an else if that catches Oracle.

Reply:
Fixed.